### PR TITLE
Changed time generated file

### PR DESCRIPTION
**Pull request by Castaldio86**
between endtime and starttime

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/SecurityEvent/RDP_RareConnection.yaml [hunk 1]
```diff
@@ -28,10 +28,10 @@ query: |
   // use left anti to exclude anything from the previous 14 days that is not rare
   | join kind=leftanti (
   SecurityEvent
-  | where TimeGenerated between (ago(starttime) .. ago(endtime))
+  | where TimeGenerated between (ago(endtime) .. ago(starttime))
```
Comment (shainw) [inline Detections/SecurityEvent/RDP_RareConnection.yaml]: Nice catch, this is a bug, but the bug is in the variable being set above in the let statement.  Change line 21 and 22 to this -
  let starttime = 14d;
  let endtime = 1d;

File: Detections/SigninLogs/FailedLogonToAzurePortal.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ requiredDataConnectors:
     dataTypes:
       - SigninLogs
 queryFrequency: 1d
-queryPeriod: 1d
+queryPeriod: 7d
```
Comment (shainw) [inline Detections/SigninLogs/FailedLogonToAzurePortal.yaml]: Thanks, we just modified this and missed making this change.

**Comment by shainw** [comment]
@Castaldio86 - Looking for your review of my comments

**Comment by shainw** [comment]
@Castaldio86 - one more time and if I don't see a change come thru, I will post a PR for the issue you brought up and link back to yours.  Thanks, Shain

**Comment by Castaldio86** [comment]
Sorry haven't noticed your comments, it was a crazy week.

**Pull request closed without merging**